packages/integration-test-app/windows/integrationtest/TestTransaction.cpp

```diff
@@ -3,10 +3,13 @@
 
 #include "pch.h"
 #include "TestTransaction.h"
+#include "ExceptionInfo.h"
+#include "TestCommandResponse.h"
 
 #include <Crash.h>;
 
 using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Data::Json;
```

Comment:
Just curious, on why sometimes using the fqn in the code, and here we import the namespace. Shouldn't we also import the .h?

Reply:
Distinction should be mostly headers vs source. It is discouraged to include "using namespace" in headers since they leak to header includes.

One other area is short namespaces. Cppcoreguidelines used to (and may still) recommend usage of FQN for things like "std", "winrt", etc. So "winrt::fire_and_foget" preferred to "fire_and_forget", but "JSonValue" preferred to "winrt::Windows::Data::Json::JsonValue"

If there's stuff that doesn't fit into that category, lmk. It's also very possible I copy/pasted from header to cpp and forgot to shorten.